pkg/openthread/contrib/netdev/openthread_netdev.c

```diff
@@ -83,12 +88,8 @@ static void *_openthread_event_loop(void *arg) {
     netdev_t *dev;
     msg_t msg, reply;
 
+#if defined(MODULE_LIBOPENTHREAD_CLI_FTD) || defined(MODULE_LIBOPENTHREAD_CLI_MTD)
```

Comment:
https://github.com/RIOT-OS/RIOT/pull/7149/files#diff-b7c878ba1ad0b6981cab03da75f2b1bdR26

Each USEMODULE added is defined as MODULE_<NAME_OF_MODULE>, I made use of that